memtier_benchmark.cpp

```diff
@@ -1293,6 +1290,11 @@ int main(int argc, char *argv[])
             exit(1);
         }
 
+        // Cluster mode supports only a single key commands
+        if (cfg.cluster_mode && cfg.arbitrary_commands->at(i).keys_count != 1) {
```

Comment:
@YaacovHazan we should change from `!=1` to `>1`.

Reply:
@YaacovHazan I've added the test and confirmed behaviour is as expected with the code change.